### PR TITLE
Tests, auto-update and various fixes

### PR DESCRIPTION
**Pull request by archetipico**
Better tests.
Solved #64. To accomplish it I use a cronjob that starts when the machine is rebooted.
Also when the program is updated, it no longer re-installs itself because that process was useful in the beginning, when we had a different environment and we kept changing aliases. Now we have tests, so use them to check if everything matches the updates.

### CODE REVIEW

**Comment by archetipico** [comment]
Now also solving #66

**Comment by archetipico** [comment]
Solving #73

**Comment by archetipico** [comment]
Solving #70

**Review by MasterCruelty — COMMENTED** [review]
I've seen quickly the changes. I'm gonna test it tomorrow I guess.
Is it all working for you? @archetipico

**Comment by archetipico** [comment]
> Is it all working for you? @archetipico

It is for me and for another user too

File: src/utils/cron.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+fi
+
+# Initialize crontab
+sudo crontab 2>/dev/null
+
+# Add line to crontab
+ROOT=${SRC::-3}
+JOB="@reboot source ${ROOT}update.sh $ROOT 2>>$ROOT.errors"
+( sudo crontab -u $USER -l; echo $JOB ) | sudo crontab -u $USER - 2>/dev/null
+printProgress "Cronjob successfully installed $COOL"
```
Comment (MasterCruelty) [inline src/utils/cron.sh:32]: @archetipico here I added a couple of lines to display the crontab installed to the user

**Review by MasterCruelty — COMMENTED** [review]
👓

**Review by MasterCruelty — APPROVED** [review]
Added a couple of line on ```cron.sh```

**Comment by MasterCruelty** [comment]
> Added a couple of line on `cron.sh`

I'm gonna merge

**Pull request merged**